kong/api/routes/plugins.lua

```diff
@@ -94,20 +130,14 @@ do
         out[k] = v
       end
     end
+    if is_array == "yes" or is_array == "maybe" then
+      setmetatable(out, cjson.empty_array_mt)
```

Comment:
Consider using `array_mt` if possible. The issues with `empty_array_mt` were partially described in my other comment, but overall, it makes for unpredictable results (or hard to predict) since this will leave tables to be encoded as an Object or an Array solely depending on whether they are empty or not... It could be that the `maybe` branch isn't sure how to encode the table, but we better be certain if possible. The same way, the table could deserve to be encoded as an empty Object in some cases, while we are making it an empty Array with this metatable. And vice-versa if the table _has_ fields.